fastapi/dependencies/utils.py

```diff
@@ -1,5 +1,6 @@
 import asyncio
 import inspect
+import logging
```

Comment:
I saw there's a logging module already in fastapi, would that be the correct one to use?